src/ensembl/src/shared/state/genome/genomeActions.ts

```diff
@@ -108,10 +101,11 @@ export const fetchGenomeTrackCategories: ActionCreator<ThunkAction<
 
     dispatch(fetchGenomeTrackCategoriesAsyncActions.request(genomeId));
 
-    const url = `/api/genomesearch/genome/track_categories?genome_id=${genomeId}`;
+    const url = `/api/tracks/track_categories/${genomeId}`;
     const response = await apiService.fetch(url);
-    updatedGenomeTrackCategories[genomeId] = response.track_categories;
 
+    updatedGenomeTrackCategories[genomeId] =
+      response.track_categories.reverse();
```

Comment:
Let's talk to the backend team about this.

Reply:
@veidenberg will update the track api to return the correct order of categories

Reply:
I have updated the category order in the backend side ([example request](http://staging-2020.ensembl.org/api/tracks/track_categories/homo_sapiens_GCA_000001405_28)).